solr/packaging/test/test_healthcheck.bats

```diff
@@ -29,8 +29,9 @@ teardown() {
 }
 
 @test "healthcheck on cloud solr" {
-  solr start -c -e films
-  run solr healthcheck -c films
+  solr start -c -e films -p ${SOLR_PORT}
+  run solr healthcheck -c films -zkhost localhost:${ZK_PORT} -V
   refute_output --partial 'error'
```

Comment:
Did you say that the refute should be case insensitive? Is that even supported in BATS?

Reply:
so, when I added a `--partial 'ERROR'` it didn't help because we actually put that message out in a legitimate run!   So I backed out that change.   There is a path where the healthcheck fails at the Java layer, and writes out "ERROR", and we don't see that due to our refute.   I guess a better approach is to look for positive things ;-)